Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
```

Comment:
There were a few places in the project files where we needed a common mapping from build-time target frameworks to what the TPSDK-conventions are.  So I declared them here, and this file is picked up and included by every project file underneath this directory.